### ISSUE TITLE
Please add support for Angular 10

### ISSUE THREAD

**Issue by SimonWilliams-STL** [webcat12345/ngx-ui-switch#412]
When updating a project to Angular 10 I get:
```
npm WARN ngx-ui-switch@9.0.2 requires a peer of @angular/core@>=5.0.0 <10.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-ui-switch@9.0.2 requires a peer of @angular/common@>=5.0.0 <10.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-ui-switch@9.0.2 requires a peer of @angular/forms@>=5.0.0 <10.0.0 but none is installed. You must install peer dependencies yourself.
```
My limited testing indicates that the package works okay with Angular 10 and so it may be as simple as updating the dependencies in package.json.

**Comment by cmckni3** [comment]
Correct, Angular 10.x is supported because of Angular's backwards compatibility model since Angular 4.x.

**Comment by cmckni3** [comment]
Published 10.0.0
